src/Root.tsx

```diff
@@ -0,0 +1,21 @@
+import { HashRouter as Router, Route, Routes } from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/pages/HomePage';
+import { PeoplePage } from './components/pages/PeoplePage';
+import { NotFoundPage } from './components/pages/NotFoundPage';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route element={<App />}>
+          <Route path="/:home?" element={<HomePage />} />
+          <Route path="/people" element={<PeoplePage />}>
+            <Route path=":slug" element={<PeoplePage />} />
```

Comment:
It seems like you are trying to nest the same component (`PeoplePage`) within itself. This might lead to an infinite loop of component rendering, which is a critical issue. If you were trying to handle a specific route (like `/people/:slug`), you should create a new component for that route and use it here instead of `PeoplePage`.